connector_node/risingwave-connector-service/src/main/java/com/risingwave/connector/SinkValidationHandler.java

```diff
@@ -0,0 +1,36 @@
+package com.risingwave.connector;
+
+import com.risingwave.connector.api.TableSchema;
+import com.risingwave.connector.api.sink.SinkFactory;
+import com.risingwave.proto.ConnectorServiceProto;
+import com.risingwave.proto.ConnectorServiceProto.SinkConfig;
+import io.grpc.stub.StreamObserver;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class SinkValidationHandler {
+    private final StreamObserver<ConnectorServiceProto.ValidateSinkResponse> responseObserver;
+    static final Logger LOG = LoggerFactory.getLogger(SinkValidationHandler.class);
+
+    public SinkValidationHandler(
+            StreamObserver<ConnectorServiceProto.ValidateSinkResponse> responseObserver) {
+        this.responseObserver = responseObserver;
+    }
+
+    public void handle(ConnectorServiceProto.ValidateSinkRequest request) {
+        try {
+            SinkConfig sinkConfig = request.getSinkConfig();
+            TableSchema tableSchema = TableSchema.fromProto(sinkConfig.getTableSchema());
+            SinkFactory sinkFactory = SinkUtils.getSinkFactory(sinkConfig.getSinkType());
+            sinkFactory.validate(tableSchema, sinkConfig.getPropertiesMap());
+        } catch (Exception e) {
```

Comment:
I think we'd better have an info log for the exception so that the exception stack trace will not be swallowed.